src/alr/alr-commands-search.adb

```diff
@@ -104,14 +104,14 @@ package body Alr.Commands.Search is
          ------------------------
 
          procedure List_All_Or_Latest
-           (Crate : Alire.Projects.With_Releases.Crate) is
+           (Crate : Alire.Crates.With_Releases.Crate) is
          begin
             if Cmd.Full then
                for Release of Crate.Releases loop
                   List_Release (Release);
                   Busy.Step;
                end loop;
-            else
+            elsif not Crate.Releases.Is_Empty then
```

Comment:
This is more than renaming. What is the reason for the change?

Reply:
Ah, good eye. It's a bug fix needed after #293, for the case where we had a crate without releases (there aren't in the current index). This PR was initially on top of my externals experiments in #295. I caught the bug then, and when rebasing this PR I didn't realize this was a functional change (even if it's a bug fix).